crates/jxl-render/src/lib.rs

```diff
@@ -437,6 +437,7 @@ impl<'f> ContextInner<'f> {
 
         if let Some(splines) = &lf_global.splines {
             let mut estimated_area = 0;
+            let image_size = (self.image_header.size.width * self.image_header.size.height) as u64;
```

Comment:
We need to check the dimension of the frame, so `frame.header().width` and `.height` would be correct here.

Reply:
Thanks! Also Replaced `&Headers` with `&FrameHeader` in `blend`